src/Symfony/Component/Validator/Tests/Constraints/NoSuspiciousCharactersValidatorTest.php

```diff
@@ -111,35 +116,43 @@ public static function provideSuspiciousStrings(): iterable
                 'restrictionLevel' => NoSuspiciousCharacters::RESTRICTION_LEVEL_MINIMAL,
                 'locales' => ['en'],
             ],
-            NoSuspiciousCharacters::RESTRICTION_LEVEL_ERROR,
-            'This value contains characters that are not allowed by the current restriction-level.',
+            [NoSuspiciousCharacters::RESTRICTION_LEVEL_ERROR => 'This value contains characters that are not allowed by the current restriction-level.'],
         ];
 
         yield 'Fails INVISIBLE check because of duplicated non-spacing mark' => [
             'à̀',
             [
                 'checks' => NoSuspiciousCharacters::CHECK_INVISIBLE,
             ],
-            NoSuspiciousCharacters::INVISIBLE_ERROR,
-            'Using invisible characters is not allowed.',
+            [NoSuspiciousCharacters::INVISIBLE_ERROR => 'Using invisible characters is not allowed.'],
         ];
 
         yield 'Fails MIXED_NUMBERS check because of different numbering systems' => [
             '8৪',
             [
                 'checks' => NoSuspiciousCharacters::CHECK_MIXED_NUMBERS,
             ],
-            NoSuspiciousCharacters::MIXED_NUMBERS_ERROR,
-            'Mixing numbers from different scripts is not allowed.',
+            [NoSuspiciousCharacters::MIXED_NUMBERS_ERROR => 'Mixing numbers from different scripts is not allowed.'],
         ];
 
         yield 'Fails HIDDEN_OVERLAY check because of hidden combining character' => [
             'i̇',
             [
                 'checks' => NoSuspiciousCharacters::CHECK_HIDDEN_OVERLAY,
             ],
-            NoSuspiciousCharacters::HIDDEN_OVERLAY_ERROR,
-            'Using hidden overlay characters is not allowed.',
+            [NoSuspiciousCharacters::HIDDEN_OVERLAY_ERROR => 'Using hidden overlay characters is not allowed.'],
+        ];
+
+        yield 'Fails both HIDDEN_OVERLAY and RESTRICTION_LEVEL checks' => [
```

Comment:
shouldn't this test be added in older branches as well (and so all the test refactoring) ?